audibot_gazebo/include/audibot_gazebo/AudibotInterfacePlugin.hpp

```diff
@@ -1,18 +1,20 @@
 #pragma once
 
 #include <rclcpp/rclcpp.hpp>
-#include <example_interfaces/msg/float64.hpp>
-#include <example_interfaces/msg/u_int8.hpp>
+#include <std_msgs/msg/float64.hpp>
+#include <std_msgs/msg/u_int8.hpp>
 #include <geometry_msgs/msg/twist_stamped.hpp>
 #include <nav_msgs/msg/odometry.hpp>
 
 #include <gazebo/common/Plugin.hh>
 #include <gazebo/common/Time.hh>
 #include <gazebo/physics/physics.hh>
 #include <gazebo_ros/node.hpp>
-#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>
```

Comment:
Only `tf2_geometry_msgs/tf2_geometry_msgs.h` exists in ROS 2 Foxy, but only `tf2_geometry_msgs/tf2_geometry_msgs.hpp` exists in ROS 2 Iron. This change breaks ROS 2 Foxy, but since it is already EOL, I think this is fine.